### PR TITLE
Cargo additions, bugfixes and changes

### PR DESCRIPTION
**Pull request by BoomerThor**
Operation Managers and Hangar Techs were supposed to always have the Cargo Control program downloadable, but it was access-locked behind HOP (which has been changed to executive officer). I also gave the program the ability to run on laptops, so cargo techs actually have a reason to bring a laptop.

Fixes some cargo bounties that were bugged (including #17225) and removed some that cannot be fulfilled. Smoke grenades were removed from security, machinists can't print the Telecomms Traffic Control board, science can't print NVGs anymore, and there is no way in-game to produce monkey hide.

### CODE REVIEW

**Comment by BoomerThor** [comment]
!review

File: code/game/objects/structures/crates_lockers/closets.dm [hunk 1]
```diff
@@ -395,6 +395,9 @@
 		if(welded)
 			to_chat(user, SPAN_WARNING("[attacking_item] detects that [src] is welded shut, and refuses to scan."))
 			return
+		playsound(src.loc, 'sound/machines/twobeep.ogg', 50, 1)
+		playsound(src.loc, 'sound/bureaucracy/print.ogg', 20, 1)
+		sleep(5)
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/crates_lockers/closets.dm:400]: Use a timer instead of a sleep

Adding beeps is also not a bugfix

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/cargo/bounties/chef.dm [hunk 2]
```diff
@@ -263,11 +270,30 @@
...
-			/obj/item/reagent_containers/food/snacks/soup/zantiri,
-			/obj/item/reagent_containers/food/snacks/xuqqil,
-			/obj/item/reagent_containers/food/snacks/qlguabi)
+	wanted_types = list(/obj/item/reagent_containers/food/snacks/)
+
+/datum/bounty/item/chef/skrell/applies_to(var/obj/item/reagent_containers/food/snacks/O)
+	if(!..())
+		return FALSE
+	if(O.icon == 'icons/obj/item/reagent_containers/food/cultural/skrell.dmi')
+		return TRUE
```
Comment (NonQueueingMatt) [inline code/modules/cargo/bounties/chef.dm:279]: checking the icon isn't an acceptable way to do this - icon files are prone to being changed and generally should never be used to check the state or type of an object

File: code/modules/cargo/bounties/chef.dm [hunk 3]
```diff
@@ -245,16 +254,14 @@
...
-			/obj/item/reagent_containers/food/snacks/stew/tajaran,
-			/obj/item/reagent_containers/food/snacks/tajaran_bread,
-			/obj/item/reagent_containers/food/snacks/tajcandy)
+	wanted_types = list(/obj/item/reagent_containers/food/snacks)
+
+/datum/bounty/item/chef/tajara/applies_to(var/obj/item/reagent_containers/food/snacks/O)
+	if(!..())
+		return FALSE
+	if(O.icon == 'icons/obj/item/reagent_containers/food/cultural/tajara.dmi')
+		return TRUE
```
Comment (NonQueueingMatt) [inline code/modules/cargo/bounties/chef.dm:263]: checking the icon isn't an acceptable way to do this - icon files are prone to being changed and generally should never be used to check the state or type of an object

File: code/modules/cargo/bounties/chef.dm [hunk 4]
```diff
@@ -226,17 +238,14 @@
...
-			/obj/item/reagent_containers/food/snacks/chilied_eggs,
-			/obj/item/reagent_containers/food/snacks/sliceable/eyebowl,
-			/obj/item/reagent_containers/food/snacks/hatchbowl)
+	wanted_types = list(/obj/item/reagent_containers/food/snacks)
+
+/datum/bounty/item/chef/unathi/applies_to(var/obj/item/reagent_containers/food/snacks/O)
+	if(!..())
+		return FALSE
+	if(O.icon == 'icons/obj/item/reagent_containers/food/cultural/unathi.dmi')
+		return TRUE
```
Comment (NonQueueingMatt) [inline code/modules/cargo/bounties/chef.dm:247]: checking the icon isn't an acceptable way to do this - icon files are prone to being changed and generally should never be used to check the state or type of an object

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**